packages/creator-hub/preload/vite.config.js

```diff
@@ -16,7 +16,7 @@ const config = {
   envDir: PROJECT_ROOT,
   resolve: {
     alias: {
-      '/shared/': join(PROJECT_ROOT, 'packages', 'shared') + '/',
+      '/shared/': join(PACKAGE_ROOT, '../shared') + '/',
```

Comment:
I think using:
```suggestion
      '/shared/': join(PROJECT_ROOT, 'shared') + '/',
```
should work